src/main/java/spotify/models/albums/AlbumFull.java

```diff
@@ -10,6 +10,30 @@
 
 import java.util.List;
 
+/**
+ * Class AlbumFull
+ * @package spotify.models.albums
+ * @author jzheng2017
+ * @version 1.0.0
+ * Provides the AlbumFull class with the following information:
```

Comment:
This is really not necessary

Reply:
Could you give me the description for this AlbumFull class. Since there are many other classes in this package, if I can get one example from you, I can use it as a reference in terms of format and type of content you want.